### PR TITLE
Feature/fgs 221

### PR DESCRIPTION
**Pull request by MatthewWaanders**

### CODE REVIEW

File: .homeycompose/drivers/settings/autoOffBehaviour.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "type": "checkbox",
+  "label": {
+    "en": "Auto-off Behaviour",
```
Comment (RobinBol) [inline .homeycompose/drivers/settings/autoOffBehaviour.json]: capitalisation

File: .homeycompose/drivers/settings/autoOffBehaviour.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    "nl": "Automatisch uit gedrag"
+  },
+  "hint": {
+    "en": "This parameter ({{zwaveParameterIndex}}) determines if pressing the switch a second time will immediately turn off the relays.",
+    "nl": "Deze parameter ({{zwaveParameterIndex}}) Bepaalt of het tweemaal indrukken van de schakelaar de relais direct uitschakelt.",
```
Comment (RobinBol) [inline .homeycompose/drivers/settings/autoOffBehaviour.json]: capitalisation

File: .homeycompose/drivers/settings/switchType.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{
+  "type": "dropdown",
+  "label": {
+    "en": "Switch Type",
```
Comment (RobinBol) [inline .homeycompose/drivers/settings/switchType.json]: capitalisation

File: .homeycompose/drivers/settings/switchType.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  "hint": {
+    "en": "This parameter ({{zwaveParameterIndex}}) determines what device is operating the switch.",
+    "nl": "Deze parameter ({{zwaveParameterIndex}}) bepaalt welk apparaat de schakelaar bestuurt.",
+  },
+  "value": "1",
+  "values": [
+    {
+      "id": "0",
+      "label": {
+        "en": "Momentary Switch",
```
Comment (RobinBol) [inline .homeycompose/drivers/settings/switchType.json]: capitalisation

File: .homeycompose/drivers/settings/switchType.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+        "nl": "Puls Schakelaar"
+      }
+    },
+    {
+      "id": "1",
+      "label": {
+        "en": "Toggle Switch",
```
Comment (RobinBol) [inline .homeycompose/drivers/settings/switchType.json]: capitalisation

File: drivers/FGS-221/device.js [hunk 6]
```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const ZwaveDevice = require('homey-meshdriver').ZwaveDevice;
+
+class FibaroDoubleSwitchDevice extends ZwaveDevice {
+	
+	onMeshInit() {
+		if(!this.node.isMultiChannelNode) {
```
Comment (RobinBol) [inline drivers/FGS-221/device.js]: Weird indentation

File: drivers/FGS-221/driver.compose.json [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+  "id": "FGS-221",
+  "name": {
+    "en": "Double Relay Switch"
+  },
+  "class": "socket",
+  "capabilities": [
+    "onoff"
+  ],
+  "images": {
+    "large": "${{driverAssetsPath}}/images/large.png",
```
Comment (RobinBol) [inline drivers/FGS-221/driver.compose.json]: $ sign again?

File: drivers/FGS-221/driver.compose.json [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+      261,
+      262,
+      263,
+      265,
+      4106,
+      8202,
+      12298
+    ],
+    "learnmode": {
+      "image": "${{driverAssetsPath}}/learnmode.svg",
```
Comment (RobinBol) [inline drivers/FGS-221/driver.compose.json]: Also here

File: drivers/FGS-221/driver.compose.json [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+      "instruction": {
+        "en": "Press the button on your Fibaro Double Relay Switch three times",
```
Comment (RobinBol) [inline drivers/FGS-221/driver.compose.json]: Use deviceNameEn/Nl

File: drivers/FGS-221/driver.compose.json [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+    "associationGroups": [
+      3
+    ],
+    "multiChannelNodes": {
+      "2": {
+        "class": "socket",
+        "capabilities": [
+          "onoff"
+        ],
+        "icon": "${{driverAssetsPath}}/icon-2.svg",
```
Comment (RobinBol) [inline drivers/FGS-221/driver.compose.json]: Again

File: drivers/FGS-221/driver.settings.compose.json [hunk 11]
```diff
@@ -0,0 +1,53 @@
+[
```
Comment (RobinBol) [inline drivers/FGS-221/driver.settings.compose.json]: Are these all settings of the device?

File: drivers/FGS-221/driver.settings.compose.json [hunk 12]
```diff
@@ -0,0 +1,53 @@
...
+  {
+    "$extends": "switchType",
+    "id": "switch_type",
+    "zwave": {
+      "index": 14,
+      "size": 1
+    }
+  },
+  {
+    "id": "save_power_state",
```
Comment (RobinBol) [inline drivers/FGS-221/driver.settings.compose.json]: You probably want to extend this one, pretty common setting, the label is now a bit weird.

File: drivers/FGS-221/driver.settings.compose.json (hunk 11 above)
Comment (MatthewWaanders) [reply drivers/FGS-221/driver.settings.compose.json]: Yes, these were all the settings included in the previous version of the application

File: .homeycompose/drivers/settings/switchType.json (hunk 3 above)
Comment (MatthewWaanders) [reply .homeycompose/drivers/settings/switchType.json]: Fixed, possibly commented on the wrong line

**Pull request merged**